pay-api/setup.cfg

```diff
@@ -71,6 +71,7 @@ notes=FIXME,XXX,TODO
 ignored-modules=flask_sqlalchemy,sqlalchemy,SQLAlchemy,alembic,scoped_session
 ignored-classes=scoped_session
 min-similarity-lines=15
+max-attributes=15
```

Comment:
I think 7 was a bit too low, 15 is reasonable.. for some of our larger search dataclasses